cmd/clusterctl/client/cluster/mover.go

```diff
@@ -647,18 +654,16 @@ func waitReadyForMove(ctx context.Context, proxy Proxy, nodes []*node, dryRun bo
 		}
 		key := client.ObjectKeyFromObject(obj)
 
-		blockLogged := false
-		if err := retryWithExponentialBackoff(ctx, backoff, func(ctx context.Context) error {
-			if err := c.Get(ctx, key, obj); err != nil {
+		log.Info(fmt.Sprintf("Move blocked by %s annotation, waiting for it to be removed", clusterctlv1.BlockMoveAnnotation))
+		if err := retryWithExponentialBackoff(ctx, waitForResourceMoveUnblockedBackoff, func(ctx context.Context) error {
+			if err := retryWithExponentialBackoff(ctx, getResourceBackoff, func(ctx context.Context) error {
```

Comment:
Is `getResourceBackoff` basically an interval to retry under the long `waitForResourceMoveUnblockedBackoff`? I think a comment to explain this may help. I'm just a little confused about the nested `retryWithExponentialBackoff`.